config/locales/en.yml

```diff
@@ -2767,6 +2767,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       signup_or_login: "Start By Signing Up (or logging in)"
       have_an_account: "Already have an account?"
       action_login: "Log in now."
+    stripe_elements:
+      unknown_error_from_stripe: |
+        Uh oh, the Stripe integration failed. You can try again but there is
+        a very small chance that the payment came through already. It's best to
+        check your bank account first.
```

Comment:
The user is checking out with their card and may have no knowledge of what is Stripe.
This message is in the catch, submit was not executed.
This createToken is only creating the card in stripe, not making the payment (that's done on server side checkout, no call to submit means no server side checkout).
So, I'd make this error message a bit different:
"There was a problem setting up your card in our payments gateway. Please refresh the page and try again, if it fails a second time, please contact us for support."

Reply:
Ah, yes, right. I updated that with your text.